src/java/org/apache/cassandra/fql/FullQueryLogger.java

```diff
@@ -117,6 +136,19 @@ public synchronized void enable(Path path, String rollCycle, boolean blocking, i
         }
     }
 
+    public FullQueryLoggerOptions getFullQueryLoggerOptions()
+    {
+        if (isEnabled())
+        {
+            return fullQueryLoggerOptions;
```

Comment:
the options should pull from `BinLog` rather than trying to store the previous `FullQueryLoggerOptions` options; `binLog` field is the source of truth for FQL